### PR TITLE
chore(structure): Remove unused file, rework folder structure and add image snapshots

### PR DESCRIPTION
**Pull request by bdauria**
This PR is a major cleanup of centreon-ui. 

It removes all unused Components and reworks folder structure to regroup all parts (test, story, ...) of a Component within the same directory:

Screenshot from 2019-12-22 13-40-10

All stories have been converted into CSF format, and are now covered automatically by image snapshot testing. This now makes manual snapshot testing obsolete, so such test files have been removed.

A new script has been added to build the Storybook statically: `npm run build:storybook`. This command must be executed before running image snapshot tests locally (via `npm t`)

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -18,8 +18,5 @@ const withThemeProvider = (story) => (
 
 addDecorator(withThemeProvider);
 
-function loadStories() {
-  require('../stories/index.js');
-}
+configure(require.context('../src', true, /\.stories\.jsx?$/), module);
```
[the remark this answers is not in the document]
Comment (bdauria) [reply .storybook/config.js]: We might need to fine tune using this parameter, depending on the results we get from the CI.
I'll have to revisit the setup on Jenkins with the help of @ganoze

File: install-chrome-headless.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+#!/bin/bash
+# from https://chromium.woolyss.com/
+# and https://gist.github.com/addyosmani/5336747
+# and https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md
+sudo apt-get update
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:canonical-chromium-builds/stage
+sudo apt-get update
+sudo apt-get install chromium-browser
```
[the remark this answers is not in the document]
Comment (bdauria) [reply install-chrome-headless.sh]: The CI fails at the moment, I'll check it out

[the remark this answers is not in the document]
Comment (bdauria) [reply install-chrome-headless.sh]: fixed by using the puppeteer docker image, with chrome installed by default

File: .storybook/config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bdauria) [reply .storybook/config.js]: @Thebarda turned out the threshold was needed, along with a delay before capturing the screenshots to take animations into account.

File: src/Button/Save/index.jsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ const SaveButton = ({ succeeded, loading, tooltipLabel, ...rest }) => {
     }
 
     if (loading) {
-      return <CircularProgress size={30} />;
+      return <CircularProgress size={30} color="white" />;
```
Comment (kduret) [inline src/Button/Save/index.jsx]: Do we want it white ?

File: src/InputField/index.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import InputField from '.';
+
+export default { title: 'InputField' };
+
+export const smalWithTitle = () => (
```
Comment (kduret) [inline src/InputField/index.stories.jsx]:
```suggestion
export const smallWithTitle = () => (
```

File: src/Button/Save/index.jsx (hunk 3 above)
Comment (bdauria) [reply src/Button/Save/index.jsx]: Yes, that's how we use it in BAM. We might want to personalize depending on future uses

**Pull request merged**